From 0bc030c417adfa7ca50944c918101dd9ce62bebb Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@debian.org>
Date: Fri, 24 Jul 2015 17:17:58 +0200
Subject: [PATCH 1/2] Update information about DEB_TARGET_* variables

These are used to support building cross-compilers. Introduced in
dpkg 1.17.14.
---
 policy.sgml | 16 ++++++++++++----
 1 file changed, 12 insertions(+), 4 deletions(-)

diff --git a/policy.sgml b/policy.sgml
index 404dc73..72a2505 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -2175,9 +2175,16 @@ zope.
 	  the architecture on which <file>debian/rules</file> is run and
 	  the package build is performed.  The host architecture is the
 	  architecture on which the resulting package will be installed
-	  and run.  These are normally the same, but may be different in
+	  and run.  The target architecture is the architecture of the
+	  packages that the compiler currently being built will generate.
+	  These are normally the same, but may be different in
 	  the case of cross-compilation (building packages for one
-	  architecture on machines of a different architecture).
+	  architecture on machines of a different architecture), building a
+	  cross-compiler (a compiler package that will generate objects for
+	  one architecture, built on a machine of a different architecture)
+	  or a Canadian cross-compiler (a compiler that will generate
+	  objects for one architecture, built on a machine of a different
+	  architecture, that will run on yet a different architecture).
 	</p>
 
 	<p>
@@ -2205,8 +2212,9 @@ zope.
 		<tt>DEB_*_GNU_TYPE</tt>)
 	  </list>
 	  where <tt>*</tt> is either <tt>BUILD</tt> for specification of
-	  the build architecture or <tt>HOST</tt> for specification of the
-	  host architecture.
+	  the build architecture, <tt>HOST</tt> for specification of the
+	  host architecture or <tt>TARGET</tt> for specification of the
+	  target architecture.
 	</p>
 
 	<p>
-- 
2.5.0.rc2.392.g76e840b

